iphone/cli/commands/_build.js

```diff
@@ -4409,6 +4411,71 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 			writeAssetContentsFile.call(this, path.join(launchImageDir, 'Contents.json'), launchImageSet);
 		},
 
+		function createAssetImageSets() {
+			this.logger.info(__('Creating assets image set'));
+			var assetCatalog = path.join(this.buildDir, 'Assets.xcassets'),
+				imageSets = {},
+				imageNameRegExp = /([\w\-\.\/]+)(@\dx)?(~iphone|~ipad)?\./; //[imageSetName][@Nx][~device]
```

Comment:
What images do we support? PNG? JPG? Others? We should explicitly match them in the regex.